vtk/run_test.py

```diff
@@ -12,3 +12,8 @@
 import vtk.vtkViewsPython
 import vtk.vtkVolumeRenderingPython
 import vtk.vtkWidgetsPython
+
+# Qt extension
+import vtk.QVTKWidget
+import vtk.QVTKWidget2
+import vtk.vtkQImageToImageSource
```

Comment:
if `pyqt` is not added as a runtime dependency, then I think these tests are going to fail.